CHANGELOG.md

```diff
@@ -10,7 +10,8 @@
 * MapboxNavigation now depends on MapboxNavigationNative v62.0.0. ([#3248](https://github.com/mapbox/mapbox-navigation-ios/pull/3248))
 * MapboxNavigation now depends on MapboxCommon v16.2.0. ([#3248](https://github.com/mapbox/mapbox-navigation-ios/pull/3248))
 * MapboxNavigation now depends on MapboxMobileEvents v1.0.2. ([#3039](https://github.com/mapbox/mapbox-navigation-ios/pull/3039))
-* MapboxCoreNavigation depends on MapboxDirections v2.0.0-beta.7 but no longer depends on MapboxAccounts. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808), [#2829](https://github.com/mapbox/mapbox-navigation-ios/pull/2829), [#2837](https://github.com/mapbox/mapbox-navigation-ios/pull/2837), [#3077](https://github.com/mapbox/mapbox-navigation-ios/pull/3077), [#3182](https://github.com/mapbox/mapbox-navigation-ios/pull/3182))
+* MapboxCoreNavigation depends on MapboxDirections v2.0.0-beta.8. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808))
+* MapboxCoreNavigation no longer depends on MapboxAccounts. ([#2829](https://github.com/mapbox/mapbox-navigation-ios/pull/2829))
```

Comment:
separated this changelog point in two as Minh proposed here: https://github.com/mapbox/mapbox-navigation-ios/pull/3248/files#r687243638